src/Engine/GUI/Workspaces/OrbitalWorkspace.cpp

```
@@ -185,12 +200,53 @@ void OrbitalWorkspace::loadWorkspaceConfig(const std::string &configPath) {
+		);
+		if (!selected || selected == "0" || strlen(selected) == 0)
+			return;
+
+
+		std::string selectedFilePath(selected);
 
+		if (selectedFilePath.empty())
+			return;
+
+		if (std::filesystem::is_directory(selectedFilePath)) {
+			std::filesystem::remove(selectedFilePath);
```

Clang (14.0.6)

[warning] variable 'selectedFilePath' is not initialized

(cppcoreguidelines-init-variables)